docs/processing_information.md

```diff
@@ -175,3 +175,22 @@ The [decoy-aware reference transcriptome](https://salmon.readthedocs.io/en/lates
 
 A benefit of using `salmon` is the ability to incorporate RNA-seq specific technical biases and correct counts accordingly.
 We chose to enable the [`--seqBias`](https://salmon.readthedocs.io/en/latest/salmon.html#seqbias) and [`--gcBias`](https://salmon.readthedocs.io/en/latest/salmon.html#gcbias) flags, to correct for sequence-specific biases due to random hexamer primer and fragment-level GC biases, respectively.
+
+## Merged objects
+
+In addition to downloadable objects containing a single library, we also offer a merged object to download for each project, representing all libraries contained in a single file.
+This section describes how these merged objects were prepared.
+
+Merged objects were created from the processed `SingleCellExperiment` objects for each ScPCA project.
+**These merged objects were not batch-corrected; they do not represent integrated objects.**
+
+If at least one library in the given project contained ADT data from CITE-seq experiments, the associated ADT "alternative experiment" was also merged.
+Any libraries in the given project which did not contain ADT data will contain `NA` values in the merged `counts` and `logcounts` matrices.
```

Comment:
```suggestion
Any libraries in the given project which did not contain ADT data will contain `NA` values in the merged gene by counts matrices.
```
Again, I would stray away from using terms that are specific to the SCE objects, since we haven't actually mentioned `counts` or `logcounts` in this doc.